go.mod

```diff
@@ -41,9 +40,11 @@ require (
 	google.golang.org/appengine v1.6.6 // indirect
 	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013
 	google.golang.org/grpc v1.28.0
-	k8s.io/api v0.19.2
-	k8s.io/apimachinery v0.19.2
-	k8s.io/client-go v0.19.2
-	k8s.io/code-generator v0.19.2
+	k8s.io/api v0.20.0
+	k8s.io/apimachinery v0.20.0
+	k8s.io/client-go v0.20.0
+	k8s.io/code-generator v0.20.0
```

Comment:
This was required when I added the replace for `github.com/openshift/api` below.

I believe the other updates came in transitively through the replace and these updates

Reply:
yeah, seems like you got caught in the middle of some changes